cfme/containers/container.py

```diff
@@ -59,21 +61,45 @@ def get_random_instances(cls, provider, count=1, appliance=None):
                 for obj in itertools.islice(containers_list, count)]
 
 
+class ContainerAllView(BaseLoggedInPage):
+
+    containers = Table(locator="//div[@id='list_grid']//table")
+
+    @View.nested
+    class filters(Accordion):  # noqa
+
+        ACCORDION_NAME = "Filters"
+
+        @View.nested
+        class navigation(VerticalNavigation):
+
+            DIV_LINKS_MATCHING = './/div/ul/li/a[contains(text(), {txt})]'
```

Comment:
@mshriver I really prefer to merge this ASAP with a TODO because this PR fix blocker for ~50 tests in our CI (all the tests that have navigation to this page).